src/util/versionstore.cpp

```diff
@@ -1,5 +1,6 @@
 #include "util/versionstore.h"
 
+#include <qglobal.h>
```

Comment:
The Qt docs refer to this as header by `<QtGlobal>` which already included below. https://doc.qt.io/qt-5/qtglobal.html

```suggestion
```

Reply:
Whoops, clangd is doing funny stuff again, thanks for catching this